src/sparseml/pytorch/sparsification/quantization/quantize.py

```diff
@@ -232,16 +309,26 @@ def convert_module_qat_from_schemes(module: Module):
     configure_module_default_qconfigs(module)
 
     # set modules with proper qconfigs to QAT mode
-    mapping = _get_qat_module_mappings()
+    convert_kwargs = (
+        dict(convert_custom_config_dict={})  # do not let torch override any qconfigs
+        if version.parse(torch.__version__) >= version.parse("1.8.0")
+        else {}
+    )
     torch_quantization.convert(
-        module, mapping=mapping, inplace=True, remove_qconfig=False
+        module,
+        mapping=_get_qat_module_mappings(),
+        inplace=True,
+        remove_qconfig=False,
+        **convert_kwargs,
     )
-    torch_quantization.add_observer_(module, non_leaf_module_list=set(mapping.values()))
+    # re-attach any quantization schemes lost during conversion
+    _reattach_quantization_schemes(module)
```

Comment:
Is it okay that this was moved to before the addition of observers intead of after like before?

Reply:
yup, doesn't affect anything, had kept convert and add observer together previously since they were both called under the hood under `torch.quantization.prepare_qat`, but now we need the schemes re-attached for the updated add observer

Reply:
Gotcha, nice!